src/libraries/Common/tests/System/Net/Http/HttpProtocolTests.cs

```diff
@@ -481,7 +491,12 @@ public async Task GetAsync_Chunked_VaryingSizeChunks_ReceivedCorrectly(int maxCh
                 {
                     await connection.ReadRequestHeaderAsync();
 
-                    await connection.Writer.WriteAsync($"HTTP/1.1 200 OK{lineEnding}Transfer-Encoding: chunked{lineEnding}{lineEnding}");
+                    string text = $"HTTP/1.1 200 OK{lineEnding}Transfer-Encoding: chunked{lineEnding}{lineEnding}";
+#if !NETFRAMEWORK
+                    await connection.Writer.WriteAsync(text);
+#else
+                    await connection.Writer.WriteAsync(text.ToCharArray(), 0, text.Length);
```

Comment:
I see you added an extension method for this; can this `#if` be removed?

Reply:
Yes, it's a leftover.